CONTRIBUTING.md

```diff
@@ -0,0 +1,379 @@
+# Contributing to the BIDS-matlab
+
+**Welcome to the BIDS-matlab repository!**
```

Comment:
Is the canonical styling "BIDS-matlab" or "BIDS-MATLAB"? I've seen it as "BIDS-MATLAB" other places in this repo, and MathWorks styles it "MATLAB".

Reply:
I honestly have no idea and no strong opinion about this.

We can go with `BIDS-MATLAB`.

Reply:
Will do a search and replace on the whole PR.